code/modules/clothing/spacesuits/breaches.dm

```diff
@@ -25,14 +25,20 @@
 		)
 
 /obj/item/clothing/suit/space
-
 	var/can_breach = 1                      // Set to 0 to disregard all breaching.
 	var/list/breaches = list()              // Breach datum container.
 	var/resilience = 0.2                    // Multiplier that turns damage into breach class. 1 is 100% of damage to breach, 0.1 is 10%. 0.2 -> 50 brute/burn damage to cause 10 breach damage
 	var/breach_threshold = 3                // Min damage before a breach is possible. Damage is subtracted by this amount, it determines the "hardness" of the suit.
 	var/damage = 0                          // Current total damage. Does not count patched breaches.
 	var/brute_damage = 0                    // Specifically brute damage. Includes patched punctures.
 	var/burn_damage = 0                     // Specifically burn damage. Includes patched burns.
+	origin_tech = "{'materials':3, 'engineering':3}"
+	material = MAT_PLASTIC
+	matter = list(
```

Comment:
that's a pretty odd place to add matter vars at, why not at `code\modules\clothing\spacesuits\spacesuits.dm`?

Reply:
We spoke about making clothing printable, so added it to base /suit

Reply:
At /obj/item/clothing/suit/space? What's odd about it?

Reply:
because it's not its real definition, but a partial override for breach specific vars?

Reply:
That's crazy! Disadv to using search to find definitions sometimes.